main.go

```diff
@@ -65,10 +66,12 @@ type ActiveTask struct {
 	ElapsedTime       float64  `xml:"elapsed_time"`
 	WorkingSetSize    float64  `xml:"working_set_size"`
 	ProgressRate      float64  `xml:"progress_rate"`
+	FrictionDone      float64  `xml:"fraction_done"`
```

Comment:
Friction vs fraction? :)

Reply:
yes, my bad :) 
I will fix it. Thank you!